madrigal415/hmk-4/99-bottles-revisited/bottlesOfBeer.js

```diff
@@ -0,0 +1,19 @@
+function myFunction() {
+    for (var i = 99; i>0; i--) {
+        var node = document.createElement("LI");
```

Comment:
This is minor and it makes no difference to the browser, but best practice dictates html attributes be lowercase. 
`<li></li>`